src/Components/CreateImageWizard/CreateImageWizard.tsx

```diff
@@ -187,6 +187,7 @@ const CreateImageWizard = ({ isEdit }: CreateImageWizardProps) => {
   // Firstboot
   const firstBootValidation = useFirstBootValidation();
   // Details
+  const [detailsPristine, setDetailsPristine] = useState(true);
```

Comment:
maybe change the name to 'detailsValidation'?

Reply:
This is following the naming we have going on for `filesystemPristine` a few lines above. Validation is handled via the `useValidation` hook, this variable holds information about the pristinity of the step only.